caramel/scripts/autosign.py

```diff
@@ -110,11 +111,10 @@ def main():
     del valid
 
     try:
-        certname = settings["ca.cert"]
-        keyname = settings["ca.key"]
-    except KeyError:
-        error_out("config file needs ca.cert and ca.key properly set", closer)
-    ca = models.SigningCert.from_files(certname, keyname)
+        ca_cert, ca_key = config.get_ca_cert_key_path(args, settings)
```

Comment:
The name "ca_cert"  doesn't match the content. adding _path would help. Especially for code where one could expect "ca_key" and "ca_cert" to actually contain certificates.